pylinalg/vector.py

```diff
@@ -350,7 +350,7 @@ def vector_angle_between(vector_a, vector_b, /, *, out=None, dtype=None):
     return out
 
 
-def vector_from_matrix_position(homogeneous_matrix, /, *, out=None, dtype=None):
+def mat_decompose_translation(homogeneous_matrix, /, *, out=None, dtype=None):
```

Comment:
We could also choose`mat_decompose_position`. You probably have a better idea what makes the most sense. Commenting to make sure that this a conscious decision. Also see `mat_from_translation`.

Reply:
I think translation is better. A matrix may not be directly determining a position.